From 1c06845fe3a1369ae9dc7afde0399e277db77ee7 Mon Sep 17 00:00:00 2001
From: Sachin Prabhu <sprabhu@redhat.com>
Date: Mon, 28 Jul 2014 12:16:57 -0400
Subject: [fs] cifs/netmisc: convert printk to pr_foo()

Message-id: <1406549822-10376-65-git-send-email-sprabhu@redhat.com>
Patchwork-id: 86736
O-Subject: [RHEL7.1 PATCH 64/69] [bz1082049] fs/cifs/netmisc.c: convert printk to pr_foo()
Bugzilla: 1082049
RH-Acked-by: J. Bruce Fields <bfields@redhat.com>
RH-Acked-by: Steve Dickson <SteveD@redhat.com>

From: Fabian Frederick <fabf@skynet.be>

Also fixes array checkpatch warning and converts it to static const
(suggested by Joe Perches).

Cc: Joe Perches <joe@perches.com>
Cc: Steve French <sfrench@samba.org>
Cc: Andrew Morton <akpm@linux-foundation.org>
Signed-off-by: Fabian Frederick <fabf@skynet.be>
Signed-off-by: Steve French <smfrench@gmail.com>
(cherry picked from commit 179d61839bafa924ff6f1960436339abdc911650)
Signed-off-by: Sachin Prabhu <sprabhu@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/fs/cifs/netmisc.c b/fs/cifs/netmisc.c
index 0498845..6834b9c 100644
--- a/fs/cifs/netmisc.c
+++ b/fs/cifs/netmisc.c
@@ -795,8 +795,8 @@ cifs_print_status(__u32 status_code)
  while (nt_errs[idx].nt_errstr != NULL) {
   if (((nt_errs[idx].nt_errcode) & 0xFFFFFF) ==
       (status_code & 0xFFFFFF)) {
-   printk(KERN_NOTICE "Status code returned 0x%08x %s\n",
-       status_code, nt_errs[idx].nt_errstr);
+   pr_notice("Status code returned 0x%08x %s\n",
+      status_code, nt_errs[idx].nt_errstr);
   }
   idx++;
  }
@@ -941,8 +941,9 @@ cifs_UnixTimeToNT(struct timespec t)
  return (u64) t.tv_sec * 10000000 + t.tv_nsec/100 + NTFS_TIME_OFFSET;
 }
 
-static int total_days_of_prev_months[] =
-{0, 31, 59, 90, 120, 151, 181, 212, 243, 273, 304, 334};
+static const int total_days_of_prev_months[] = {
+ 0, 31, 59, 90, 120, 151, 181, 212, 243, 273, 304, 334
+};
 
 struct timespec cnvrtDosUnixTm(__le16 le_date, __le16 le_time, int offset)
 {
-- 
1.7.1